client/ayon_core/hosts/unreal/ue_workers.py

```diff
@@ -287,11 +287,27 @@ def execute(self):
             msg = f"Unreal Python not found at {python_path}"
             self.failed.emit(msg, 1)
             raise RuntimeError(msg)
-        pyside_cmd = [python_path.as_posix(),
-                      "-m",
-                      "pip",
-                      "install",
-                      "pyside2"]
+
+        pyside_version = "PySide2"
+        ue_version = self.ue_version.split(".")
+        if int(ue_version[0]) == 5 and int(ue_version[1]) >= 4:
```

Comment:
I don't think we need to extend it to any future major Unreal versions because if they even happen, they will very likely break a lot of thinks. Also - this won't be needed when #111 is done.